integrations/docker/images/chip-build/version

```diff
@@ -1 +1 @@
-0.7.18 Version bump reason: vscode ZEPHYR SDK env path update
+0.7.19.1 Version bump reason: Reducing size of imx
```

Comment:
Why not just 0.7.19 or 0.7.20?

Reply:
To not conflict with another going on from Andrei

Reply:
For the future I would vote for 3 digit only, to make our regex history easier. Conflicts are ok and intended.